example/chain/README.md

```
@@ -102,60 +101,64 @@ not just a connectivity probe.
 | Rule | Container | Trigger | Status |
 |---|---|---|---|
 | R0001 Unexpected process launched | redis | comm=`perl` | DETECTED |
-| R0005 DNS Anomalies in container | redis | DNS query per encoded chunk | **BLIND** — `networkEventsStreaming: disable` |
-| R0011 Unexpected Egress Network Traffic | redis | UDP/53 + external resolver | **BLIND** — same |
+| R0005 DNS Anomalies in container | redis | DNS query per encoded chunk to `*.1.1.1.1.nip.io` | DETECTED |
 
 ### Coverage summary
 
 Basic chain (4 stages, default `./scripts/local-ci-chain.sh`):
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)